overrides/netlify.toml

```diff
@@ -55,6 +56,7 @@
         https://script-staging.wiz.gov.sg/widget.js
         wss://*.zendesk.com
         wss://*.zopim.com
+        https://*.dcube.cloud/
```

Comment:
this didn't have the `/` in the instructions https://docs.wogaa.sg/web-implementation/setup-guide/information-service-implementation-test/#step-3-content-security-policies-for-staging

Reply:
Going to try with the `/` first! if it doesn't work i'll switch it back